pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1311,6 +1311,8 @@ public void deleteOfflineTable(String tableName) {
     if (!_helixAdmin.getResourcesInCluster(_helixClusterName).contains(offlineTableName)) {
```

Comment:
I understand this is not part of this PR, but do we need to flag an error if table name is not found?